### PR TITLE
Spctr-1912 Fix: Translation for default text was missing for some blocks.

### PR DESCRIPTION
**Pull request by jamyrbsf**
### Description
https://brainstormforce.atlassian.net/browse/SPCTR-1912

### Screenshots
Not avalable 

### Types of changes
Changes in attributes.js in info to make the default translatable for loco translate.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,6 +205,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Resolved Buttons should consider button text as a string instead of html in multiple blocks.
 * Fix: Container - Does not work properly when it is used inside cover block.
 * Fix: Post Carousel - On reducing or increasing Post per page all items sticks to each other in editor.
+* Fix: Info Box - Some translation waas missing for some blocks.
```
Comment (RajkiranBagal) [inline README.md:208]: Please add a proper changelog. 

- Check for spelling mistakes.
- You have written Info Box at the start & further it's written for some blocks, which is confusing.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jamyrbsf) [reply README.md]: Added a proper changelog.

Comment (RajkiranBagal) [reply README.md]: @jamyrbsf I see changes in Call to Action Block & not Info Box. Am I missing something? 

Also Update PR title.

File: src/blocks/call-to-action/attributes.js [hunk 2]
```diff
@@ -14,7 +15,7 @@ const attributes = {
 	ctaTitle: {
 		source: 'html',
 		selector: 'h1,h2,h3,h4,h5,h6',
-		default: 'Call To Action',
+		default: __( 'Call To Action' ),
```
Comment (AbhijitNage123) [inline src/blocks/call-to-action/attributes.js:18]: Text-domain is missing. Translation format should be like this - `__( 'Call To Action', 'ultimate-addons-for-gutenberg' )`

File: src/blocks/call-to-action/attributes.js [hunk 3]
```diff
@@ -24,7 +25,7 @@ const attributes = {
 		source: 'html',
 		selector: 'p',
 		default:
-			'Click here to change this text. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut elit tellus, luctus nec ullamcorper mattis, pulvinar dapibus leo.',
+			__( 'Click here to change this text. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut elit tellus, luctus nec ullamcorper mattis, pulvinar dapibus leo.' ),
```
Comment (AbhijitNage123) [inline src/blocks/call-to-action/attributes.js:28]: Text-domain is missing.

File: src/blocks/call-to-action/attributes.js [hunk 4]
```diff
@@ -334,7 +335,7 @@ const attributes = {
 	},
 	ctaText: {
 		type: 'html',
-		default: 'Read More',
+		default: __( 'Read More' ),
```
Comment (AbhijitNage123) [inline src/blocks/call-to-action/attributes.js:338]: Text-domain is missing.

File: src/blocks/call-to-action/attributes.js [hunk 5]
```diff
@@ -586,7 +587,7 @@ const attributes = {
 	},
 	secondCtaLabel: {
 		type: 'html',
-		default: 'Click Here',
+		default: __( 'Click Here' ),
```
Comment (AbhijitNage123) [inline src/blocks/call-to-action/attributes.js:590]: Text-domain is missing.

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]

**Review by AbhijitNage123 — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -205,6 +205,7 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 * Fix: Resolved Buttons should consider button text as a string instead of html in multiple blocks.
 * Fix: Container - Does not work properly when it is used inside cover block.
 * Fix: Post Carousel - On reducing or increasing Post per page all items sticks to each other in editor.
+* Fix: Info Box - Some translations were missing.
```
[the remark this answers is not in the document]
Comment (jamyrbsf) [reply README.md]: Updated this text

File: src/blocks/call-to-action/attributes.js (hunk 5 above)
Comment (jamyrbsf) [reply src/blocks/call-to-action/attributes.js]: Text-domain added

File: src/blocks/call-to-action/attributes.js (hunk 4 above)
Comment (jamyrbsf) [reply src/blocks/call-to-action/attributes.js]: Text-domain added

File: src/blocks/call-to-action/attributes.js (hunk 3 above)
Comment (jamyrbsf) [reply src/blocks/call-to-action/attributes.js]: Text-domain added

File: src/blocks/call-to-action/attributes.js (hunk 2 above)
Comment (jamyrbsf) [reply src/blocks/call-to-action/attributes.js]: Text-domain added

File: README.md (hunk 1 above)
Comment (jamyrbsf) [reply README.md]: I have updated the pr name, and added text-domain and also updated changelog

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**